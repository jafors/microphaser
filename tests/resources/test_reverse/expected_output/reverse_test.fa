>a324f17d17b218dR
cGAGCCACACCCATCACTCAACATGTC
>c258647e023393eR
TTCcGAGCCACACCCATCACTCAACAT
>172c41a2e6e2cb7R
TTCTTCcGAGCCACACCCATCACTCAA
>b5f424dc651bf48R
CTTTTCTTCcGAGCCACACCCATCACT
>965e45175515136R
GATCTTTTCTTCcGAGCCACACCCATC
>08add1803a50f62R
TGAGATCTTTTCTTCcGAGCCACACCC
>223bd8fb09288ccR
GACTGAGATCTTTTCTTCcGAGCCACA
>732c0f3ff28e5c0R
AGGGACTGAGATCTTTTCTTCcGAGCC
>58c2ffd43739c73R
GGAAGGGACTGAGATCTTTTCTTCcGA
>742163ac1cebc12R
CcGCTGCCTGCCCACCACACGGCCCTC
>fd764a5345f7473R
CATCcGCTGCCTGCCCACCACACGGCC
>a55d1f2587eae5cR
GTGCATCcGCTGCCTGCCCACCACACG
>ae580a513507433R
CTTGTGCATCcGCTGCCTGCCCACCAC
>946de2980abc10fR
GGCCTTGTGCATCcGCTGCCTGCCCAC
>4d2f7903355cae3R
CAAGGCCTTGTGCATCcGCTGCCTGCC
>8a68b9eec214f8bR
GTCCAAGGCCTTGTGCATCcGCTGCCT
>56239f5beba4a30R
CCTGTCCAAGGCCTTGTGCATCcGCTG
>c8cd1f33758a3fcR
CACCCTGTCCAAGGCCTTGTGCATCcG
>0b4e37d09934bdeR
acAgggacAggggccgggaccgggacc
>0bf5651d15c846cR
acAgggactggggccgggaccgggaca
>5829bfd284f0cb5R
gggacAgggacAggggccgggaccggg
>683eeee967ad8d0R
gggacAgggactggggccgggaccggg
>4087657e230f3a5R
accgggacAgggacAggggccgggacc
>7023ad3880e2d57R
accgggacAgggactggggccgggacc
>8c3434d759140efR
gggaccgggacAgggactggggccggg
>ae280e232d0582aR
gggaccgggacAgggacAggggccggg
>6f5b9cce754f8f2R
gggaccgggacAgggactggggccggg
>561f80087272f47R
gccgggaccgggacAgggactggggcc
>e1b098d6a1588e2R
gccgggaccgggacAgggacAggggcc
>e9da669b9cab341R
ggAgccgggaccgggacAgggactggg
>4e047277cdeb322R
ggggccgggaccgggacAgggacAggg
>fc553bd6250d877R
actggAgccgggaccgggacAgggact
>f561469dc0d1bbdR
actggggccgggaccgggacAgggacA
>8aaf4c7d467b4f4R
gggactggggccgggaccgggacAggg
>cf58ccde40a93c6R
gggactggAgccgggaccgggacAggg
>18fa98f4b970791R
accgggactggggccgggaccgggacA
>f7a0e246701c5ddR
acAgggactggggccgggaccgggacA
>d6d3ec26afc54dbR
acAgggactggAgccgggaccgggacA
