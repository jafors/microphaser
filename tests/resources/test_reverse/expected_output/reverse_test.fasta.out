>62cb2647a87e5dfR
TGAGCCACACCCATCACTCAACATGTC
>a324f17d17b218dR
cGAGCCACACCCATCACTCAACATGTC
>6fb08ec7c7f47bcR
TTCTGAGCCACACCCATCACTCAACAT
>c258647e023393eR
TTCcGAGCCACACCCATCACTCAACAT
>694cc3c630d8067R
TTCTTCTGAGCCACACCCATCACTCAA
>172c41a2e6e2cb7R
TTCTTCcGAGCCACACCCATCACTCAA
>39562cfb831a701R
CTTTTCTTCTGAGCCACACCCATCACT
>b5f424dc651bf48R
CTTTTCTTCcGAGCCACACCCATCACT
>0d1064b66ef872eR
GATCTTTTCTTCTGAGCCACACCCATC
>965e45175515136R
GATCTTTTCTTCcGAGCCACACCCATC
>9dfd4a2a3e06a03R
TGAGATCTTTTCTTCTGAGCCACACCC
>08add1803a50f62R
TGAGATCTTTTCTTCcGAGCCACACCC
>1890ce6eb553e2eR
GACTGAGATCTTTTCTTCTGAGCCACA
>223bd8fb09288ccR
GACTGAGATCTTTTCTTCcGAGCCACA
>bbcca5e5926c6b8R
AGGGACTGAGATCTTTTCTTCTGAGCC
>732c0f3ff28e5c0R
AGGGACTGAGATCTTTTCTTCcGAGCC
>4b3b6ad8c866d79R
GGAAGGGACTGAGATCTTTTCTTCTGA
>58c2ffd43739c73R
GGAAGGGACTGAGATCTTTTCTTCcGA
>c9376de4a4eeb36R
CAGCTCACTAAGATCCTGCAGGCTGGC
>1c060adfd690c42R
CAaCTCACTAAGATCCTGCAGGCTGGC
>70f4032847e8b9cR
CACCAGCTCACTAAGATCCTGCAGGCT
>8584b7aeb9d2f26R
CACCAaCTCACTAAGATCCTGCAGGCT
>89e88d210eb44afR
TACCACCAGCTCACTAAGATCCTGCAG
>a5528732feff657R
TACCACCAaCTCACTAAGATCCTGCAG
>5467b8035befff8R
CTCTACCACCAGCTCACTAAGATCCTG
>33bdff14891109fR
CTCTACCACCAaCTCACTAAGATCCTG
>20ad676ed0eeb1fR
ACCCTCTACCACCAGCTCACTAAGATC
>1e6e5cfb1cdc9deR
ACCCTCTACCACCAaCTCACTAAGATC
>63b7186258fc02dR
GTCACCCTCTACCACCAGCTCACTAAG
>86aae2a3ec5fcadR
GTCACCCTCTACCACCAaCTCACTAAG
>3f7096abd8bf649R
GGTGTCACCCTCTACCACCAGCTCACT
>caa2a4b00ce540eR
GGTGTCACCCTCTACCACCAaCTCACT
>caf61cdec0e89e5R
TTGGGTGTCACCCTCTACCACCAGCTC
>5aaa3507d1d1348R
TTGGGTGTCACCCTCTACCACCAaCTC
>36dee4a50c2493eR
CACTTGGGTGTCACCCTCTACCACCAG
>c204eec32d41624R
CACTTGGGTGTCACCCTCTACCACCAa
>742163ac1cebc12R
CcGCTGCCTGCCCACCACACGGCCCTC
>fd764a5345f7473R
CATCcGCTGCCTGCCCACCACACGGCC
>a55d1f2587eae5cR
GTGCATCcGCTGCCTGCCCACCACACG
>ae580a513507433R
CTTGTGCATCcGCTGCCTGCCCACCAC
>946de2980abc10fR
GGCCTTGTGCATCcGCTGCCTGCCCAC
>4d2f7903355cae3R
CAAGGCCTTGTGCATCcGCTGCCTGCC
>8a68b9eec214f8bR
GTCCAAGGCCTTGTGCATCcGCTGCCT
>56239f5beba4a30R
CCTGTCCAAGGCCTTGTGCATCcGCTG
>c8cd1f33758a3fcR
CACCCTGTCCAAGGCCTTGTGCATCcG
>577e9ef1d73e376R
TGCCGGCGTGTTGGGTGACTCCTGGCT
>6744a2c2d41f673R
TGCCGGtGTGTTGGGTGACTCCTGGCT
>a89c8ea6bd3c93eR
CGGtGTGTTGGGTGACTCCTGGCTCCT
>dabf6a620bf78c7R
tGTGTTGGGTGACTCCTGGCTCCTCTC
>c3781a7d76cdaa3R
GTTGGGTGACTCCTGGCTCCTCTCTCG
>9737f7a0eca2ce3R
GGGTGACTCCTGGCTCCTCTCTCGCTC
>eb968544fb1b9faR
TGACTCCTGGCTCCTCTCTCGCTCAGC
>b9e7cc40ef5eb1bR
CTCCTGGCTCCTCTCTCGCTCAGCGGC
>36f1ccf0236dd7eR
CTGGCTCCTCTCTCGCTCAGCGGCCAA
>3286632c566d638R
GCTCCTCTCTCGCTCAGCGGCCAAGGT
>8e681aa30658d5aR
GGGTGCCGGCGTGTTGGGTGACTCCTG
>871c2db59be2e24R
GGGTGCCGGtGTGTTGGGTGACTCCTG
>fb16dc2abaad395R
CGGGGGTGCCGGCGTGTTGGGTGACTC
>a50701944aeb40fR
CGGGGGTGCCGGtGTGTTGGGTGACTC
>e263dcf1cffd1ecR
GAGCGGGGGTGCCGGCGTGTTGGGTGA
>6cbeb500d27ab0cR
GAGCGGGGGTGCCGGtGTGTTGGGTGA
>8870fc955acfa6aR
GGAGAGCGGGGGTGCCGGCGTGTTGGG
>9f825be514211d0R
GGAGAGCGGGGGTGCCGGtGTGTTGGG
>2a0306ddf027cf8R
CATGGAGAGCGGGGGTGCCGGCGTGTT
>380d4855b31900cR
CATGGAGAGCGGGGGTGCCGGtGTGTT
>8afcef1a5734f6aR
CAGCATGGAGAGCGGGGGTGCCGGCGT
>47689d704d16372R
CAGCATGGAGAGCGGGGGTGCCGGtGT
>e45b398cee8397eR
gccgggaccgggaccgggacagggacc
>aee7752445f7ac7R
gccgggaccgggacagggaccaggacT
>06e4d6fa2d0a92cR
ggggccgggaccgggaccgggacaggg
>e153e44397d0225R
ggggccgggaccgggacagggaccagg
>086ad8ff06c33deR
actggggccgggaccgggaccgggaca
>bc7fb48dd3846c6R
actggggccgggaccgggacagggacc
>086ad8ff06c33deR
actggggccgggaccgggaccgggaca
>0f1f6e85f76e173R
gggactggggccgggaccgggaccggg
>7e52ccba441a7b0R
gggactggggccgggaccgggacaggg
>0f1f6e85f76e173R
gggactggggccgggaccgggaccggg
>1e4997b22ab45d2R
accgggactggggccgggaccgggacc
>1e4997b22ab45d2R
accgggactggggccgggaccgggacc
>c3522362d359e4bR
accgggactggggccgggaccgggaca
>c3522362d359e4bR
accgggactggggccgggaccgggaca
>9be96014cc873e3R
gggaccgggactggggccgggaccggg
>9be96014cc873e3R
gggaccgggactggggccgggaccggg
>03a8d16f9377bd0R
gggaccgggactggggccgggaccggg
>03a8d16f9377bd0R
gggaccgggactggggccgggaccggg
>5935263355d21baR
accgggaccgggactggggccgggacc
>5935263355d21baR
accgggaccgggactggggccgggacc
>544a0c9a48cd79fR
accgggaccgggactggggccgggacc
>544a0c9a48cd79fR
accgggaccgggactggggccgggacc
>fd0f95bc205cdf2R
gggaccgggaccgggactggggccggg
>6ff07d3f62e4a2cR
gggaccgggaccgggactggggccggg
>fd0f95bc205cdf2R
gggaccgggaccgggactggggccggg
>6ff07d3f62e4a2cR
gggaccgggaccgggactggggccggg
>6ff07d3f62e4a2cR
gggaccgggaccgggactggggccggg
>f7a8162bb062070R
gccgggaccgggaccgggactggggcc
>8c910cb425ca167R
gccgggaccgggaccgggactggggcc
>f7a8162bb062070R
gccgggaccgggaccgggactggggcc
>8c910cb425ca167R
gccgggaccgggaccgggactggggcc
>0fd5f2de5cb129bR
ggggccgggaccgggaccgggacaggg
>0fd5f2de5cb129bR
ggggccgggaccgggaccgggacaggg
>4ed34e6b57bf437R
ggggccgggaccgggaccgggactggg
>4ed34e6b57bf437R
ggggccgggaccgggaccgggactggg
>920ad7f4c1d6d7cR
actggggccgggaccgggaccgggaca
>920ad7f4c1d6d7cR
actggggccgggaccgggaccgggaca
>384d5ff5188d8ceR
actggggccgggaccgggaccgggact
>384d5ff5188d8ceR
actggggccgggaccgggaccgggact
>35401280f1deda7R
gggactggggccgggaccgggaccggg
>16d5d645d9f4205R
gggactggggccgggaccgggacaggg
>35401280f1deda7R
gggactggggccgggaccgggaccggg
>16d5d645d9f4205R
gggactggggccgggaccgggacaggg
>dfe92347736da03R
accgggactggggccgggaccgggacc
>dfe92347736da03R
accgggactggggccgggaccgggacc
>dfe92347736da03R
accgggactggggccgggaccgggacc
>dfe92347736da03R
accgggactggggccgggaccgggacc
>dfe92347736da03R
accgggactggggccgggaccgggacc
>dfe92347736da03R
accgggactggggccgggaccgggacc
>560715910a2259fR
gggaccgggactggggccgggaccggg
>560715910a2259fR
gggaccgggactggggccgggaccggg
>560715910a2259fR
gggaccgggactggggccgggaccggg
>560715910a2259fR
gggaccgggactggggccgggaccggg
>f762b9aff44b7baR
accgggaccgggactggggccgggacc
>f762b9aff44b7baR
accgggaccgggactggggccgggacc
>f762b9aff44b7baR
accgggaccgggactggggccgggacc
>f762b9aff44b7baR
accgggaccgggactggggccgggacc
>ed4c5d4ddac6d98R
gggaccgggaccgggactggggccggg
>ed4c5d4ddac6d98R
gggaccgggaccgggactggggccggg
>ed4c5d4ddac6d98R
gggaccgggaccgggactggggccggg
>ed4c5d4ddac6d98R
gggaccgggaccgggactggggccggg
>076975fa347c701R
gccgggaccgggaccgggactggggcc
>076975fa347c701R
gccgggaccgggaccgggactggggcc
>076975fa347c701R
gccgggaccgggaccgggactggggcc
>076975fa347c701R
gccgggaccgggaccgggactggggcc
>828e75a3e7deeeaR
ggggccgggaccgggaccgggactggg
>828e75a3e7deeeaR
ggggccgggaccgggaccgggactggg
>828e75a3e7deeeaR
ggggccgggaccgggaccgggactggg
>828e75a3e7deeeaR
ggggccgggaccgggaccgggactggg
>bdbf32be8271a0cR
actggggccgggaccgggaccgggact
>4ed424b0372a61eR
actggggccgggaccgggaccgggaca
>7b456d8781464bdR
gggactggggccgggaccgggaccggg
>168ebadb6cb6e9dR
gggactggggccgggaccgggaccggg
>45531e958a7133eR
acagggactggggccgggaccgggacc
>3f9f2ad69ae3ebbR
acagggactggggccgggaccgggacc
